### DISCUSSION TITLE
Docker container storage filling up. How do I keep the logs from filling up the storage?

### DISCUSSION THREAD

**Discussion by andy24r** [brianschrameck/arlo-cam-api#13] [General]
I have already resized the container storage once. How can I prevent this from recurring?

**Comment by brianschrameck** [comment]
The logs go to stdout or stderr and get picked up by Docker's logging driver. You would need to configure your Docker daemon to rotate the logs.

You could do this [globally for all containers](https://docs.docker.com/engine/logging/configure/#configure-the-default-logging-driver), or for a [single container](https://docs.docker.com/engine/logging/configure/#configure-the-logging-driver-for-a-container). If you are running using Compose you can configure using [the logging element](https://docs.docker.com/reference/compose-file/services/#logging) on your service. The default driver is [json-file](https://docs.docker.com/engine/logging/drivers/json-file/#options).
